src/little-bang-ast/little_bang_ast.ml

```diff
@@ -39,6 +39,11 @@ type expr =
   | Let_expr of ast_uid * var * expr * expr
   | Appl_expr of ast_uid * expr * expr
   | Builtin_expr of ast_uid * Tiny_bang_ast.builtin_op * expr list
+  (* This is an expression in little bang, but a value in tiny bang.
```

Comment:
Does this need so much clarification?  It's no different from `Label_expr`, is it?